src/web/index.html

```diff
@@ -4,7 +4,7 @@
   <title>TEAMMATES - Online Peer Feedback/Evaluation System for Student Team Projects</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src 'self' https://www.google.com/recaptcha/ https://www.gstatic.com/recaptcha/ https://cdn.jsdelivr.net; style-src 'self' 'unsafe-inline' https://cdn.jsdelivr.net; frame-src 'self' docs.google.com https://www.google.com/recaptcha/; img-src 'self' data: http: https:; font-src 'self' https://cdn.jsdelivr.net; connect-src 'self' ws: http://localhost:*; form-action 'none'; base-uri 'self'">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src 'self' https://www.google.com/recaptcha/ https://www.gstatic.com/recaptcha/ https://cdn.jsdelivr.net https://apis.google.com; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://cdn.jsdelivr.net; frame-src 'self' docs.google.com https://www.google.com/recaptcha/ https://*.firebaseapp.com; img-src 'self' data: http: https:; font-src 'self' https://cdn.jsdelivr.net https://fonts.gstatic.com; connect-src 'self' https://*.googleapis.com ws: http://localhost:*; form-action 'none'; base-uri 'self'">
```

Comment:
The changes here should be applied to the backend as well. https://github.com/TEAMMATES/teammates/blob/604d4de96767f860dea62417841b1c0ed1c08509/src/main/java/teammates/ui/servlets/WebSecurityHeaderFilter.java#L25-L37

Reply:
I noticed that all URLs in `WebSecurityHeaderFilter.java` end with a `/` but in `index.html`, some end with '/' while some don't, e.g. `https://cdn.jsdelivr.net` instead of `https://cdn.jsdelivr.net/`. May I know if it's a convention/best practice to put a `/` at the end, or it's just a preference?

Reply:
It's largely preference, and also situational.